cinema/urls.py

```diff
@@ -1 +1,26 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register(
+    "movie_sessions",
+    MovieSessionViewSet,
+    basename="movie_session"
+)
+
+urlpatterns = [path("", include(router.urls))]
+
```

Comment:
When basepath is empty no need to include, instead use:
urlpatterns = router.urls

Reply:
Uploaded to my brain ;)